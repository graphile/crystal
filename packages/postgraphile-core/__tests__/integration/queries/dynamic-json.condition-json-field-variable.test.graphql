query myQuery($myVal: String = "test", $myVal2: JSON = {stringField: "test"}) {
  # All three of these should result in the same value
  a: allMyTables(condition: {jsonData: {stringField: $myVal}}) {
    nodes {
      id
      jsonData
    }
  }
  b: allMyTables(condition: {jsonData: {stringField: "test"}}) {
    nodes {
      id
      jsonData
    }
  }
  c: allMyTables(condition: {jsonData: $myVal2}) {
    nodes {
      id
      jsonData
    }
  }
}
