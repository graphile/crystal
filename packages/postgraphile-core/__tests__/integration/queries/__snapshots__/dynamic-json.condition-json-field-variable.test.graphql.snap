// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "a": Object {
      "nodes": Array [
        Object {
          "id": 1,
          "jsonData": Object {
            "stringField": "test",
          },
        },
      ],
    },
    "b": Object {
      "nodes": Array [
        Object {
          "id": 1,
          "jsonData": Object {
            "stringField": "test",
          },
        },
      ],
    },
    "c": Object {
      "nodes": Array [
        Object {
          "id": 1,
          "jsonData": Object {
            "stringField": "test",
          },
        },
      ],
    },
  },
}
`;
